### PR TITLE
[meta] Update issue templates

### PR DESCRIPTION
**Pull request by mcdurdin**
Here are some default issue templates that I suggest we try using for 3rd parties to submit helpful issue reports. There are a bunch of repos that do a lot more than this -- and we could learn from them -- so am happy to tweak if anyone else has ideas.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/137946:id/kvk/index.html)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (darcywong00) [inline .github/ISSUE_TEMPLATE/bug_report.md]: wouldn't Desktop OS examples be `[e.g. Windows 10 or macOS]`?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (darcywong00) [inline .github/ISSUE_TEMPLATE/bug_report.md]: is this Browser Version?

**Comment by darcywong00** [comment]
For both templates, can you add a section for the Keyman product [e.g. Desktop, Developer, etc]?

**Comment by jahorton** [comment]
I think it should be possible to create multiple templates, rather than just a single 'bug report' for all our platforms:

* https://help.github.com/articles/creating-issue-templates-for-your-repository/
  - Note:  there's a "Custom Template" option, which should be useful for defining extras.
* https://help.github.com/articles/about-issue-and-pull-request-templates/

Keep in mind that we've got a number of potential bug source types:

* "Keyboard Development" (Developer)
  - Affects keyboard coding, compilation, and debug-testing... not sure what specifics to ask here.
* Desktop
  - interactions with OS, application (not necesssarily browser)
* "Website" (KeymanWeb)
  - interactions with browser, possibly OS - but it's important we know it's KeymanWeb (browser code) and not a desktop platform (native code)
* "Smartphone" (Mobile - iOS/Android)
  - interactions with OS, app (b/c system keyboard)
  - 'app' could simply be "Keyman" for in-app issues.

I think it'd be best if these four had their own individual templates.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mcdurdin) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Tidied up

**Comment by mcdurdin** [comment]
I've made tweaks to the issue templates to better match our products. For now, I've stuck with just two templates although I agree we could usefully split out per platform in the future. I'd like to see if this improves bug reports and watch the friction on it a little before doing much more.

**Pull request merged**